site_root/input/resources/profiles/StructureDefinition-kfdrc-condition.json

```diff
@@ -0,0 +1,54 @@
+{
+    "resourceType": "StructureDefinition",
+    "id": "kfdrc-condition",
+    "url": "http://fhir.kids-first.io/StructureDefinition/kfdrc-condition",
+    "version": "0.1.0",
+    "name": "kfdrc-condition",
+    "title": "Kids First DRC Condition",
+    "status": "draft",
+    "fhirVersion": "4.0.0",
+    "kind": "resource",
+    "abstract": false,
+    "type": "Condition",
+    "baseDefinition": "http://hl7.org/fhir/StructureDefinition/Condition",
+    "derivation": "constraint",
+    "differential": {
+      "element": [
+        {
+          "id": "Condition",
+          "path": "Condition"
+        },
+        {
+          "id": "Condition.extension",
+          "path": "Condition.extension"
+        },
+        {
+          "id": "Condition.extension:age-at-event",
+          "path": "Condition.extension",
+          "sliceName": "age-at-event",
+          "sliceIsConstraining": false,
+          "label": "age_at_event",
+          "short": "Patient age at determination",
+          "definition": "Age of the Patient when their condition was registered",
+          "min": 0,
+          "max": "1",
+          "type": [
+            {
+              "code": "Extension",
+              "profile": [
+                "http://fhir.kids-first.io/StructureDefinition/age-at-event"
+              ]
+            }
+          ],
+          "mustSupport": true,
+          "isModifier": false
```

Comment:
Not necessary. See: https://www.hl7.org/fhir/elementdefinition-definitions.html#ElementDefinition.isModifier

Also, if an extension is being used as a modifier, it needs to be profiled on the "modifierExtension", not "extension".  The reason is that a system that doesn't have access to a profile but gets an extension (i.e. doesn't know the meaning and can't tell if it is a modifier or not) in a data instance, that instance needs a fool proof way for knowing if it's a modifier or not. That's why there is the "modifierExtension". Any extension in data under "modifierExtension" has to be understood by the server, or the record should probably be ignored.  see: http://hl7.org/fhir/domainresource-definitions.html#DomainResource.modifierExtension